diffpy/pdfmorph/pdfplot.py

```diff
@@ -139,55 +139,55 @@ def comparePDFs(pairlist, labels=None, rmin = None, rmax = None, show = True,
     scale = min(1, max(scale, 0.8))
     figsize = [13.5, 4.5]
     figsize[0] *=  scale
-    fig = pylab.figure(1, figsize = figsize)
+    fig = plt.figure(1, figsize = figsize)
     # Get the margins based on the figure size
     lm = 0.12 / scale
     bm = 0.20 / scale
     rm = 0.02 / scale
     tm = 0.15 / scale
-    axes = pylab.Axes(fig, [lm, bm, 1 - lm - rm, 1 - bm - tm])
+    axes = plt.Axes(fig, [lm, bm, 1 - lm - rm, 1 - bm - tm])
     fig.add_axes(axes)
-    pylab.minorticks_on()
+    plt.minorticks_on()
 
-    pylab.plot(rdat, grdat, label = labeldata, marker = 'o', markerfacecolor
+    plt.plot(rdat, grdat, label = labeldata, marker = 'o', markerfacecolor
             = 'white', markeredgecolor = 'blue', markersize = 7,
             markeredgewidth = 0.75)
-    pylab.plot(rfit, grfit, label = labelfit, linestyle = 'solid', linewidth =
+    plt.plot(rfit, grfit, label = labelfit, linestyle = 'solid', linewidth =
             2, color = 'red')
-    pylab.plot(rdat, offset*numpy.ones_like(diff), linestyle = '--', linewidth
+    plt.plot(rdat, offset*numpy.ones_like(diff), linestyle = '--', linewidth
             = 1, color = 'black', dashes = (15, 15), aa = False)
     diff += offset
-    pylab.plot(rdat, diff, label = labeldiff, linestyle = 'solid',
+    plt.plot(rdat, diff, label = labeldiff, linestyle = 'solid',
             linewidth = 1.5, color = 'green')
 
     if maglim is not None:
         # Add a line for the magnification cutoff
-        pylab.axvline(maglim, 0, 1, linestyle = '--', color = 'black',
+        plt.axvline(maglim, 0, 1, linestyle = '--', color = 'black',
                 linewidth = 1.5, dashes = (14, 7))
         # FIXME - look for a place to put the maglim
         xpos = (rvmax*0.85 + maglim) / 2 / (rvmax - rvmin)
         if xpos <= 0.9:
-            pylab.figtext(xpos, 0.7, "x%.1f"%mag, backgroundcolor='w')
+            plt.figtext(xpos, 0.7, "x%.1f"%mag, backgroundcolor='w')
 
     # Get a tight view
-    pylab.xlim(rvmin, rvmax)
+    plt.xlim(rvmin, rvmax)
     ymin = min(diff[sel])
     ymax = max(max(grdat[sel]), max(gtemp[sel]))
     yspan = ymax - ymin
-    # Give a small boarder to the plot
+    # Give a small border to the plot
     gap = 0.05 * yspan
     ymin -= gap
     ymax += gap
-    pylab.ylim(ymin, ymax)
+    plt.ylim(ymin, ymax)
 
     # Make labels and legends
-    pylab.xlabel("r $(\AA)$")
-    pylab.ylabel("G $(\AA^{-1})$")
-    #pylab.legend(loc = 0)
+    plt.xlabel("r $(\AA)$")
```

Comment:
I think this will give italic angstrom labels.  Not sure how to fix, maybe ``$\mathrm(\AA)$``?

Reply:
I think I just solved it along with the commented code! Hopefully all is well

Reply:
Closed the other PR and did the changes in the right branch hopefully